TasksApp/TasksApp/ViewControllers/ContainerViewController.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  ContainerViewController.swift
+//  TasksApp
+//
+//  Created by user on 03.04.2024.
+//
+
+import UIKit
+
+final class ContainerViewController: UIViewController {
+    
+    enum MenuState {
+        case closed
+        case opened
+    }
+    
+    // MARK: - Private Properties
```

Comment:
comments are a good thing, but excessive comments - not. I can see that those variables are private, we don't need a comment for that

Reply:
fixed